src/fparser/scripts/fparser2.py

```diff
@@ -68,6 +68,9 @@
 from __future__ import print_function
 from fparser.scripts.script_options import set_fparser_options
 
+import logging
```

Comment:
pylint complains that this import is in the wrong place.

Reply:
Quite right. Apologies.